src/main/java/frc/utils/battery/BatteryLimiter.java

```diff
@@ -3,18 +3,44 @@
 import edu.wpi.first.math.filter.LinearFilter;
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Robot;
+import frc.robot.constants.IPs;
+import frc.utils.CMDHandler;
 import frc.robot.constants.GlobalConstants;
 import frc.utils.DriverStationUtils;
+import frc.utils.dashboard.LoggedTableBoolean;
 import org.littletonrobotics.junction.Logger;
 
+import java.nio.file.Path;
+
 class BatteryLimiter extends Command {
 
     private static final int NUMBER_OF_SAMPLES_TAKEN_IN_AVERAGE = 50;
+    private static final double MESSAGE_STARTUP_TIME = 0.2; // 2 python cycles
```

Comment:
I would prefer to split this to 2 constants of python cycle time and number of cycles to wait.

Also, what do you mean by python cycle time?

Reply:
The loop in python runs every 100 ms

Reply:
And we check connection to robot and then check if we should show the message, so we need to wait 2 cycles for it to surely be able to show message before stopping